app/assets/javascripts/short-form/directives/preference-proof-uploader.html.slim

```diff
@@ -11,7 +11,7 @@
     | {{'D2_LIVE_WORK_PREFERENCE.LETTER_FROM_EMPLOYER_INSTRUCTIONS' | translate}}
 
 .form-group.text-center ng-if="show_preference_uploader()"
-  button.button.ghost-white.expand ngf-select="uploadProof($file, preference, application.preferences[preference_proof_option])" ngf-accept="'image/jpg, image/jpeg, image/png, application/pdf'" name="file_{{preference_proof_file}}" ngf-max-size="2MB" ng-required="application.preferences[preference]" aria-describedby="{{preference_proof_file}}_error" ng-model="application.preferences[pref_type_proof_file]"
+  button.button.ghost-white.expand ngf-select="uploadProof($file, preference, application.preferences[preference_proof_option])" ngf-accept="'image/jpg, image/jpeg, image/png, application/pdf'" name="file_{{preference_proof_file}}" ngf-max-size="2MB" ng-required="application.preferences[preference]" aria-describedby="{{preference_proof_file}}_error" ng-model="application.preferences[preference_proof_file]"
```

Comment:
@chrisdolendo note #1: remember we had to re-add `ng-model` here? well `pref_type_proof_file` didn't seem to be anything any more, so my hunch is that correcting this might fix our "uploaded file is sometimes blank" issue though still unclear. But this at least has the correct var name now.